<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FixForSSL</class>
 <widget class="QMainWindow" name="FixForSSL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>First Time Setup</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="head_label">
      <property name="text">
       <string>The CWE program is unable to locate SSL libraries on your machine.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>If you like, you can instruct the CWE program to retrive the (free) OpenSSL libraries, and install them in the CWE program folder. Then, restart the CWE program.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_license">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>View OpenSSL license at:
https://www.openssl.org/source/license.html</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_install">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Download OpenSSL dll from:
https://indy.fulgan.com/SSL/openssl-1.0.2k-x64_86-win64.zip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_exit">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Exit
(Choose this to install SSL yourself)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>(Please note: Due to various import and export restrictions, we cannot package cryptographic SSL libraries with the CWE program. Please insure you are abiding by all regulations in your area before downloading these libraries.)
SimCenter is not in any way affliated with OpenSLL or the above linked websites.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>857</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_exit</sender>
   <signal>clicked()</signal>
   <receiver>FixForSSL</receiver>
   <slot>exitProgram()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>730</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>FixForSSL</receiver>
   <slot>exitProgram()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_install</sender>
   <signal>clicked()</signal>
   <receiver>FixForSSL</receiver>
   <slot>getSSL()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_license</sender>
   <signal>clicked()</signal>
   <receiver>FixForSSL</receiver>
   <slot>showLicense()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>exitProgram()</slot>
  <slot>showLicense()</slot>
  <slot>getSSL()</slot>
 </slots>
</ui>
